### ISSUE TITLE
Contributor guide: parallelizing pytest

### ISSUE THREAD

**Issue by BGR360** [pola-rs/polars#14150]
### Description

Quoting from https://docs.pola.rs/development/contributing/test/#running-unit-tests:

> Tests can be run in parallel using [pytext-xdist](https://pytest-xdist.readthedocs.io/en/latest/). Run `pytest -n auto` to parallelize your test run.

As a mainly-Rust developer, it's not clear what steps I need to take to "use pytext-xdist". The linked docs suggest to use `pip install`, but I also know that `polars` uses a virtual environment to do stuff, and so I would think there's a different/better way to do it than using pip.

It would be nice to have just a quick sentence on this page to clarify how one should configure `pytext-xdist` to be used.

### Link

https://docs.pola.rs/development/contributing/test/#running-unit-tests
